components/table/table/Table.jsx

```diff
@@ -54,18 +56,39 @@ const Table = (props) => {
 		hideDisabledPages,
 		onFilterClear,
 		v2,
+		search,
```

Comment:
`search` and `setSearch` should not be props. It can be local state. pass prop `onSearch` that will be called on click of the search button,

Reply:
`customSearchIcon` and `onSearch` can be passed inside singleProps called searchProps as below:

```
searchProps = {
   onSearch:()=>{},
   icon: <Icon />
}
```